CHANGELOG.md

```diff
@@ -13,6 +13,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Filter_pvsyst method filters on IL Pmin, IL Pmax, IL Vmin, and IL Vmax and warns if any of the four are missing. Previously failed if any of the four were missing.
 - cp_results returns a warning if the regression formulas of the passed CapData objects do not match instead of warning and continuing.
+- pvlib 0.6.3 is required; there are issues introduced by the pvlib 0.7.0 release not yet addressed.
```

Comment:
@bt- is pvlib v0.7 compatibility something you'd like help with?

Reply:
@cwhanse, Yes, that would be great!

I imagine it will be straightforward to fix.  If my memory is correct, the test that failed the Travis CI build with pvlib 0.7 was `test_pvlib_system` when it called https://github.com/pvcaptest/pvcaptest/blob/4df1283d336d2cd7009fc51501e58cfe8795b191/captest/capdata.py#L622

If it would be helpful, I can add a file to the repo to create a conda environment that will run the tests.  The existing `environment.yml` file does not include pytest and possibly a few other packages used in testing.

Reply:
Looks like the failure is caused by the name changes in the SAM inverter file. Change [this ](https://github.com/pvcaptest/pvcaptest/blob/4df1283d336d2cd7009fc51501e58cfe8795b191/captest/capdata.py#L657) line to

`    cec_inverter = cec_inverters['ABB__MICRO_0_25_I_OUTD_US_208__208V_']`

I think pvlib v0.7 will work for you then.

Reply:
@cwhanse, thanks for the help. It looks like that change did fix that error, but there were other additional errors.  I am going to tackle updates for pvlib 0.7.0 on a separate pull request.  I created #44 to track that.

Reply:
Sorry about the trouble. I can't run the tests locally (ran into some build error for sklearn), if you want to set up a branch with the tests run on travis I'm happy to help with issues.

Reply:
No problem! I was working on setting up a branch for this and I realized, if you are willing to work on the updates for pvlib 0.7 compatibility, that it will be easier if you create a branch and pull request.  Travis will automatically run all the tests once you make a pull request.  I'm not seeing that you have a fork of the project, so that would be the first step.

Please forgive me if you are clear on the step-by-step already.  I'm going to write it out here in case it is helpful and so I can reference this and add it to the formal documentation later.

1. Fork pvcaptest in github
2. Clone fork to your computer
3. Create a new branch locally, I was going to name the branch pvlib070
4. Make revisions and commit them to the new branch
5. Push the new branch to your fork on github `git push -u origin branch_name`
6. Navigate to the new branch on your fork on github and there will be a button you can click to make a pull request
7. Once you submit the pull request Travis will run tests and the coverage report will be updated

Reply:
I'm familiar with the process but a recipe and reminders are always appreciated!